inc/packages/namespace.php

```diff
@@ -13,7 +13,7 @@
 
 const SERVICE_ID = 'FairPackageManagementRepo';
 const CONTENT_TYPE = 'application/json+fair';
-const DID_CACHE_LIFETIME = 5 * MINUTE_IN_SECONDS;
+const CACHE_LIFETIME = 5 * MINUTE_IN_SECONDS;
```

Comment:
Someone smarter than me probably thought about this 5 minute cache lifetime, but... Doesn't that seem short? If we can cache something for 5 minutes, we _could_ also cache for, an hour, two hours? 12 hours?

Reply:
6 or 12 hours is probably sufficient, but I would like more input.